particle/architecture/component-structure.md

```diff
@@ -77,3 +75,28 @@ export function enable($context) {
 export default enable;
 ```
 
+## Using Demos
+
+Every component has an optional demo folder, which is automatically pulled into Pattern Lab. These files are not built to destination apps at all, so you should never do work in them that you expect to see in a final product.
```

Comment:
"These files are only built for Pattern Lab, so you should never do work in them that you expect to see on a production app, like a Drupal theme"

Reply:
More succinct *and* informative.